contracts/pools/PoolCollection.sol

```diff
@@ -564,6 +576,53 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
         emit DepositLimitUpdated({ pool: pool, prevDepositLimit: prevDepositLimit, newDepositLimit: newDepositLimit });
     }
 
+    /**
+     * @dev reduces the trading liquidity of a given pool
+     *
+     * requirements:
+     *
+     * - the caller must be the owner of the contract
+     */
+    function reduceTradingLiquidity(Token pool, uint256 bntAmount) external onlyOwner {
+        Pool storage data = _poolStorage(pool);
+        uint256 bntTradingLiquidity = data.liquidity.bntTradingLiquidity;
+
+        if (bntAmount >= bntTradingLiquidity) {
+            return;
+        }
+
+        if (bntAmount == 0) {
+            _resetTradingLiquidity(bytes32(0), pool, data, TRADING_STATUS_UPDATE_ADMIN);
+            return;
+        }
+
+        if (bntAmount < _networkSettings.poolFundingLimit(pool)) {
```

Comment:
would be helpful to have some comments throughout the code to provide the rationale for these conditions, especially this one

Reply:
Done